src/tray/SDL_tray_utils.c

```diff
@@ -0,0 +1,62 @@
+/*
+  Simple DirectMedia Layer
+  Copyright (C) 1997-2024 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+#include "SDL_internal.h"
+
+#include "../video/SDL_sysvideo.h"
+#include "../events/SDL_events_c.h"
+
+static long long active_trays = 0;
```

Comment:
use Uint64 instead of long log

Reply:
Sorry, I really meant `Sint64`

Reply:
BTW, why does this have to be >=64 bits? Why is a simple int not enough?

Reply:
Yes, this should be int.

Reply:
Sorry, I eagerly mimicked the pattern I made in a similar case for a much lighter object type in an unrelated project, where creating several billion instances of an object was reasonable. I'm pretty sure any sane system would run out of memory before creating 2147483647 system tray icons.

Should I use `Sint32`? The C standard requires that `int` only reaches at least up to 32767, which is an envisionable (although either erroneous or malicious) limit to be reached. I'm not sure how widespread is a 16-bit `int` across compilers and platforms, but Sint32 would provide better reliability.

Reply:
Just use int.